toffy/test_utils.py

```diff
@@ -289,6 +288,23 @@ def case_extrabin(self):
 
 
 class WatcherCases:
+    """Test cases for start_watcher
+
+    Cases in this class will, in order, return:
+        (partial per_fov callbacks, per_run callbacks, directory validation functions)
+
+    per_fov callbacks are made partial here, which allowa directory arguments to be created
+    and passed within the actual test function.  Otherwise, these test cases would have to handle
+    the mangement (or somehow be aware) of the temporary directory where the testing happens.
+
+    per_run callbacks are currently not partialed since none are implemented.  That being said,
+    they probably will undergo the same treatment once some are implemented.
+
+    Validation functions will check that the directory/files created by each callback are correct.
+    Some maybe partialed for convinience, since some arguments, like the panel, are created here
+    and might as well be premptively passed, so that only "run dependent" arguments need to be
+    passed to the validators (i.e out directory and point names).
+    """
```

Comment:
@ngreenwald lmk if this works for now. 

Tbh, `class TestCases:` docstrings should probably be required+standardized at some point, so that it's easier to read at a glance.  Maybe something like:
```
class TestCases:
    """Test cases for blah blah blah
    
    Long winded diatribe about why these test cases look they way they look

    Cases return:
        (return_types[0]):
            description for return[0]
        (return_types[1]):
            ...
    """
```

happy to change it to a format like this, but just wanted to get the info down in the meantime.

Reply:
Yeah what you have there is great